.travis.yml

```diff
@@ -23,18 +23,36 @@ install:
 matrix:
   fast_finish: true
   include:
+    # JDK 8
+    - jdk: openjdk8
+      env:
+        - DESC="test package (openjdk8)"
+        - CMD="mvn test package && git diff"
+
     # JDK 8 (minimal supported Java version of eclipse-cs)
     - jdk: openjdk8
       env:
         - DESC="install (openjdk8)"
         - CMD="mvn install && git diff"
```

Comment:
The `install` phase will run the `test` and `package` phases, so this shouldn't be necessary.